jenkins_home/jobs/serialized-irods-build-and-test-workflow/config.xml

```diff
@@ -176,57 +185,62 @@
                 [$class: &apos;StringParameterValue&apos;, name: &apos;PARAMETER_IMAGE_TAG&apos;, value: job_number],
                 [$class: &apos;StringParameterValue&apos;, name: &apos;PARAMETER_IRODS_BUILD_DIR&apos;, value: buildOutputDirectory],
                 [$class: &apos;StringParameterValue&apos;, name: &apos;PARAMETER_PLUGIN_GIT_COMMITISH&apos;, value: PARAMETER_IRODS_COMMITISH],
-                [$class: &apos;StringParameterValue&apos;, name: &apos;PARAMETER_DATABASE_TYPE&apos;, value: PARAMETER_DATABASE_TYPE]
+                [$class: &apos;StringParameterValue&apos;, name: &apos;PARAMETER_DATABASE_TYPE&apos;, value: PARAMETER_DATABASE_TYPE],
+                [$class: &apos;StringParameterValue&apos;, name: &apos;PARAMETER_MYSQL_ODBC_CONNECTOR_DIR&apos;, value: PARAMETER_MYSQL_ODBC_CONNECTOR_DIR]
             ]
         }  
     }
     if(PARAMETER_RUN_TOPOLOGY_ICAT_TESTS == &quot;true&quot;) {
-        stage(&quot;Install and Run Topology Tests on Provider&quot;) {
+        stage(&quot;Install and Run Topology Tests&quot;) {
```

Comment:
think you lost some 'on Provider'... and the three others below...

Reply:
Done